sectionsUtilityTool/README.txt

```diff
@@ -0,0 +1,18 @@
+[Sections Utility Tool]
```

Comment:
You may want to rename this file to README.md, to take advantage of Markdown syntax.